tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -262,6 +381,86 @@ static void test_tx_cb_multi(void)
 	PASS();
 }
 
+static void test_tx_proxy_mixin(void)
+{
+	static struct bt_mesh_prov prov = {
+		.uuid = test_prov_uuid,
+	};
+	int err;
+
+	/* Initialize mesh stack and enable pb gatt bearer to emit beacons. */
+	bt_mesh_device_setup(&prov, &comp);
+	err = bt_mesh_prov_enable(BT_MESH_PROV_GATT);
+	ASSERT_OK(err, "Failed to enable GATT provisioner");
+
+	/* Start pb gatt beacons with 100 msec interval */
+	k_sleep(K_MSEC(1800));
+
+	LOG_INF("Provision device under test");
+	/* Provision dut and start gatt proxy beacons. */
+	bt_mesh_provision(test_net_key, 0, 0, 0, adv_cfg.addr, adv_cfg.dev_key);
+
+	/* Start proxy beacons with 1 sec interval */
+	k_sleep(K_MSEC(6000));
+
+	/* Send adv data */
+	struct net_buf *buf = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_LOCAL_ADV,
+			BT_MESH_TRANSMIT(5, 20), K_NO_WAIT);
+	net_buf_add_mem(buf, txt_msg, sizeof(txt_msg));
+	bt_mesh_adv_send(buf, NULL, NULL);
+	k_sleep(K_MSEC(150));
+
+	/* Start proxy beacons again */
+	k_sleep(K_MSEC(6000));
+
+	PASS();
+}
+
+static void test_rx_proxy_mixin(void)
+{
+	static struct bt_mesh_test_gatt gattparam = {
+		.transmits = 1500 / 100,
```

Comment:
Then it should be clarified in the code, e.g.:
```suggestion
		.transmits = 1500 /* total adv duration */ / 100 /* interval*/,
```

This could even be macro taking number of advertisements and then used in both places, here and in the dut test function.